crates/rpc/rpc-eth-api/src/helpers/call.rs

```diff
@@ -240,16 +240,13 @@ pub trait EthCall: Call + LoadPendingBlock {
         let precompiles = get_precompiles(env.handler_cfg.spec_id);
         let mut inspector = AccessListInspector::new(initial, from, to, precompiles);
 
-        let (result, env) = match self.inspect(&mut db, env, &mut inspector) {
-            Ok((res, env)) => (res, env),
-            Err(e) => {
-                return Err(e.into());
-            }
-        };
+        let (result, env) = self.inspect(&mut db, env, &mut inspector)?;
 
         let error = match result.result {
             ExecutionResult::Halt { reason, .. } => Some(format!("{:?}", reason)),
```

Comment:
this should use + `to_string`

https://github.com/paradigmxyz/reth/blob/ef84899410b1327843721a00d8bf6cf6155cbcb7/crates/rpc/rpc-eth-types/src/error.rs#L410-L410

Reply:
@mattsse 

https://github.com/paradigmxyz/reth/blob/d7112927cf617220414f290f5ec233660f431a89/crates/rpc/rpc-eth-api/src/helpers/call.rs#L245-L249

Reply:
we still need to use 

https://github.com/paradigmxyz/reth/blob/ef84899410b1327843721a00d8bf6cf6155cbcb7/crates/rpc/rpc-eth-types/src/error.rs#L410-L410

to create the appropriate error message

Reply:
Sorry to bother you. I fix it.

https://github.com/paradigmxyz/reth/blob/49ef67974604e93395c4be0ae1fbf627466a6f4f/crates/rpc/rpc-eth-api/src/helpers/call.rs#L245-L251